odk1-src/collect-import-export.rst

```diff
@@ -1,50 +1,43 @@
-Importing and Exporting Settings
-====================================
+Configuring Collect via QR code
+================================
 
 .. image:: /img/collect-settings/import-settings.*
   :alt: Import/export settings menu of ODK Collect
   :class: device-screen-vertical
 
-QR Code
-----------
-
-QR Code or the Quick Response Code is a two dimensional barcode. QR codes can be used to configure Collect on many devices. The QR image presented contains all of your current General and Admin settings, including admin and server passwords. When this QR code is scanned from the ODK Collect app on another device, these settings are imported.
-
-Sharing QR code
+Settings QR codes
 ------------------
 
-You can click on the |share| icon to share the QR code as an image. When you click on it, it displays a list of applications and services like *whatsapp*, *facebook*, *hangouts*, *bluetooth*, *MMS* to name a few, which can be used to share the QR code. This is useful when there are several different data collection sites and all devices have to be configured in the same way, in which case the QR code can be shared from one reference device. 
+QR codes are a type of barcode that can encode a lot of data with low risk of error. These can be used to configure Collect on a fleet of devices. The QR image displayed on the :guilabel:`Configure via QR code` screen contains all of your current General and Admin settings. When this QR code is scanned from the ODK Collect app on another device, the settings from the code are imported.
 
-.. |share| image:: /img/collect-settings/share-icon.*
-             :alt: Share icon for sharing the QR code. 
-             :height: 43 px
-             :width: 43 px
+.. note::
+  QR codes generated by Collect only contain settings with non-default values because of the constraints on the amount of data a QR code can hold.
```

Comment:
I'm not sure this quite conveys the effect of not having default values included. Is it that non-default values in the app won't be overriden by the QR code?

Reply:
Yes, you're right. I've tried to improve that. The note is now redundant with the text above it but I think that's ok because it's quite an important point.